### PR TITLE
[WIP]add network and port mappings for compat layer

### PR DESCRIPTION
**Pull request by baude**
the network and port information was being skipped when creating a container using the compatibility layer.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: pkg/spec/createconfig.go [hunk 1]
```diff
@@ -403,6 +404,32 @@ func (c *CreateConfig) getContainerCreateOptions(runtime *libpod.Runtime, pod *l
 		options = append(options, libpod.WithHealthCheck(c.HealthCheck))
 		logrus.Debugf("New container has a health check")
 	}
+
+	// Add ports
+	if !c.Network.NetMode.IsHost() {
+		var ports []ocicni.PortMapping
+		for i := range c.Network.PortBindings {
```
Comment (TomSweeneyRedHat) [inline pkg/spec/createconfig.go:411]: I'm not a fan of one char variables that are used more than a line or two away.  Suggest:
```suggestion
		for binding := range c.Network.PortBindings {
```
or a longer variable name of your choosing

**Comment by mheon** [comment]
@baude What's going on with this one - do you want to rebase and strip the WIP?

**Comment by mheon** [comment]
@baude Any chance this will be ready for 2.2? Or is this abandoned?

**Comment by vrothberg** [comment]
Ping

**Comment by vrothberg** [comment]
That's working now.  I did not track down the PR/commit but tested locally.

**Pull request closed without merging**